.circleci/config.yml

```diff
@@ -28,6 +28,11 @@ references:
     - &nodejs_next "14"
     ## TODO: replace this with 16 as soon as it becomes available
     - &nodejs_experimental "15"
+    ## NOTE: the nodejs version shipped with circleci/windows@2.4.0 orb
+    ## does fail on the test-functional test related to importing
+    ## web-ext as a library, the issue seems to have been fixed on more
+    ## recent nodejs versions.
```

Comment:
Can you be more explicit about the broken/fixed versions? And add the explicit sentence "Install a recent version without this issue." (Change "recent" to something more accurate if needed)

And if you think that it helps, also include a link to this PR/comment.

Reply:
I've been a bit lazy, and so I didn't run a bisect, but in the end it would be nice to write down what was the broken range while I'm still around this issue.

Based on a quick bisect, the range between 12.11.1 and 12.15.0 (included) is broken, the first version to work as expected is v12.16.0.

I updated the comment accordingly, but I didn't looked for the first version broken, I don't think we care too much about that in the end.